docs/guides/analyzers/feature_extraction.md

```diff
@@ -2,22 +2,41 @@
 hide:
   - footer
 ---
-The feature extraction analyzer creates attributes out of event data based on regular expressions. Different
-features can be specified in the `data/regex_features.yaml` file.
+The feature extraction analyzer creates attributes out of event data based on
+different extraction plugins.
 
-Please be aware that this analyzer does *not* extract ipv4, email-addresses and similar from *all* events, but only those that match the query_string.
+Currently supported are:
+* Regular expression based extractions
+  * [Regex Extraction Plugin](#regex-extraction-plugin)
+* Plaso parsed windows event logs
+  * [Winevt Extraction Plugin](#winevt-extraction-plugin)
+
+> **Note**
+Please be aware that this analyzer does *not* extract ipv4, email-addresses and
+similar from *all* events, but only those that match the definitions configured
+for the plugins explained below!
 
 ### Use case
 
-This analyzer is helpful to built a list of `email_addresses` in a sketch that are used in in `WEBHIST`. To do that, run the analyzer to have the feature extracted. Check the results by running a query: `email_address:*`.
+This analyzer is helpful to extract additional data from events as separate
+attributes. Those extracted attributes can then be used in lookups, correlations,
```

Comment:
first thing where it could be used would be searches imho, then lookups, correlations etc.